<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminScore</class>
 <widget class="QWidget" name="AdminScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QCheckBox" name="getCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="updateCheckBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="getDoneButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить
группу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getPatternButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить
шаблон</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setExcelButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Назначить
Excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
